### DISCUSSION TITLE
Remove halls from schedule on dates when they are not available

### DISCUSSION THREAD

**Discussion by shaib** [pretalx/pretalx#1806] [Ideas] [answered]
We have a two-day conference. One day is for talks -- we have a venue with three halls. The other day is for workshops -- three different rooms, in a different venue.

If I try to make a schedule, I get a six-column table, where half the columns are empty on one day, and the other half empty on the second.

I have marked the relevant hall availability properly, and I'll get warnings if I try to put sessions in halls on the wrong day. But still, bad schedule display -- my only option is to drop the Grid display.

**Comment by rixx** [comment]
That's reasonable, but also hard to implement, with the sticky room headers that persist while scrolling down the full schedule.

For events using the widget on their own sites, we'll likely make this easier soon, by allowing to embed only specific days (or rooms!), so that organisers can choose to embed the schedule multiple times to allow for multi-column layouts.

While a more flexible overall display would be great to have, I can't currently anticipate having the time to build this – however, if anybody wants to take a stab at it, I'd be more than glad to work with and guide a PR (this would be work on our separate [schedule](/pretalx/schedule) repository).

**Comment by rixx** [comment, marked as answer]
Just wanted to say that we’ve introduced the more flexible column display a while ago, in b7afb60, which will be part of the upcoming v2025.1.0 release!

**Reply by shaib** [reply]
Thanks!
